.zed/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "formatter": "prettier"
```

Comment:
I’ve been using Zed more often; we have a `.vscode` folder so hopefully this isn’t too much noise?